CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ include(nwx_versions)
 # These are the default values, used only if the user hasn't already set them.
 option(BUILD_TESTING "Should we build the tests?" OFF)
 option(BUILD_SHARED_LIBS "Should we build shared libraries?" ON)
-option(BUILD_PYBIND11_PYBINDINGS "Should we build pybind11 python bindings?" OFF)
+option(BUILD_PYBIND11_PYBINDINGS "Should we build pybind11 python bindings?" ON)
```

Comment:
Do we want to change the default here, or should this be changed in the toolchain?

Reply:
I think the default should be the lean build without bindings.

Reply:
I'd actually argue they should be on by default. If we're expecting users to interact with NWX through the Python bindings, they shouldn't have to enable the bindings. That said, off by default until we're ready to declare the Python bindings released is fine by me.

Reply:
I think Python users would expect/prefer `pip install parallelzone` not the `cmake` build. That said, I am fine either way for the default.